--- Makefile_org	2014-07-04 09:48:18.000000000 +0200
+++ Makefile	2014-07-04 10:12:15.000000000 +0200
@@ -26,6 +26,7 @@
 VERSION=$(shell cat ../VERSION)
 DESTDIR?=/usr
 PREFIX?=/local
+PWD=`pwd`
 
 LDCONFIG?=ldconfig

 ifneq ($V,1)
 Q ?= @
 endif

 STATIC=libwiringPiDev.a
 DYNAMIC=libwiringPiDev.so.$(VERSION)
@@ -40,7 +41,7 @@
 #DEBUG	= -g -O0
 DEBUG	= -O2
 CC	= gcc
-INCLUDE	= -I.
+INCLUDE	= -I. -I$(PWD)/../wiringPi
 DEFS	= -D_GNU_SOURCE
 CFLAGS	= $(DEBUG) $(DEFS) -Wformat=2 -Wall -Winline $(INCLUDE) -pipe -fPIC
 
